### PR TITLE
fix(cart-wishlist): cart-wishlist

### PR DESCRIPTION
**Pull request by musabehonore**
# What does this PR do?
Fixes products naming on cart

[187300115]

### CODE REVIEW

**Review by RWEMAREMY — APPROVED** [review]
good

**Review by Ruthuwamahoro — APPROVED** [review]

File: src/controllers/cartsController.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export const addItemToCart = async (req: Request, res: Response) => {
       await cart.save();
       const returnCart = {
         id: cart.id,
-        products: await formatCartItems([product]),
+        produtcs: await formatCartItems([product]),
```
Comment (fab-ryan) [inline src/controllers/cartsController.ts]: why writing it wrong?

File: src/controllers/wishlistController.ts [hunk 2]
```diff
@@ -16,7 +16,6 @@ export const addToWishlist = async (req: Request, res: Response) => {
   try {
     const product = await Database.Product.findOne({
       where: { id: productId },
-      attributes: ['sellerId', 'name'],
     });
     if (!product) {
```
Comment (fab-ryan) [inline src/controllers/wishlistController.ts:20]: why removing those attriibutes

**Review by fab-ryan — CHANGES_REQUESTED** [review]
some mistake in writtings

**Review by fab-ryan — APPROVED** [review]

**Pull request merged**